Contribute/validation-reference/tabbed-conceptual.md

```diff
@@ -76,6 +76,7 @@ The following tab name/tab ID pairs are whitelisted. Dependent tab IDs are not p
 |`[.NET Core 2.2]`     |`(#tab/netcore22)`|
 |`[.NET Core 3.x]`     |`(#tab/netcore3x)`|
 |`[.NET Core 3.0]`     |`(#tab/netcore30)`|
+|`[.NET Core 3.0 Preview 1]` |`(#tab/netcore30)`|
```

Comment:
@meganbradley Are tab IDs supposed to be unique? I noticed this has the same ID as the tab above it.

Reply:
We have an open question on that. @TinaLiuGirl can you confirm whether tab IDs need to be unique?

Reply:
@meganbradley @TinaLiuGirl @scottaddie  Any update on this old open discussion?